@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardStatus.vue

```diff
@@ -1,7 +1,11 @@
 <template>
-  <UiCard>
+  <UiCard :color="hasError ? 'error' : undefined">
     <UiCardTitle>{{ $t("status") }}</UiCardTitle>
-    <template v-if="isReady">
+    <UiSpinner v-if="isLoading" class="spinner" />
+    <template v-else-if="hasError">
+      <NoData />
+    </template>
```

Comment:
No need to wrap element with `template` here

```suggestion
    <NoData v-else-if="hasError" />
```